### DISCUSSION TITLE
Unable to authenticate my GitHub when I want to Git Push

### DISCUSSION THREAD

**Discussion by SmartJoojo** [community/community#61713] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

when I try to git push my commits to my GitHub, I get this information "Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication." whenever I enter my password. The terminal does not even allow me to enter my token

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @SmartJoojo ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
